### ISSUE TITLE
[Task] Prediction Market Token Addr and Meta Info

### ISSUE THREAD

**Issue by Tedko** [DimensionDev/Maskbook#2071]
https://twitter.com/catnip_exchange/status/1336362574477029376

```
Will Donald Trump Attend the 2021 U.S. Presidential Inauguration ?

Buy YES (yINAUG token) for ~20 cents on the dollar.

Or pick up some NO (nINAUG token) for ~80 cents a pop. 
```
```
Token addresses:
yINAUG: 0x934e03e671e09528a52cb3373dc7bb7ca475b924

nINAUG: 0x55b2969a6d832c6091ba1af4ed6d0e6e7c2c0d90

iINAUG (Invalid) : 0x8ee4Fc14a3fb92E998D86c29C92097047f424BB1

Market ID: 0x6e6ffb10179febf29b0223d22793d1c1d8a8f541

```

Also https://twitter.com/catnip_exchange/status/1329130928568823810

```
Will the Dems get a “Blue Sweep” by winning the Presidency, the House, *and* the Senate?

Think so? Buy yBlue for around 20 cents on the dollar 🟦

Think not? Get some nBlue 🟥
```
For y/nBlue addr: https://twitter.com/catnip_exchange/status/1329130929466400769?s=20

**Comment by BenDavidow** [comment]
Balancer pools in case it's useful for prices:
yINAUG/nINAUG/DAI: https://pools.balancer.exchange/#/pool/0x91ffe46e57d7c3f5f6e4f0856f2a674d0d820f93/
yBlue/nBlue/DAI: https://pools.balancer.exchange/#/pool/0x68c74e157f35a3e40f1b02bba3e6e3827d534059/shares

If it's too much of a hassle to get prices without coingecko don't worry about it for now and you can just leave out prices for INAUG and BLUE. Most important thing for now is the display for yTrump and nTrump, thanks!

**Comment by BenDavidow** [comment]
token names:

yINAUG - YES Trump at Inauguration Token
nINAUG - NO Trump at Inauguration Token
yBLUE - YES Dem Control Token
nBLUE - NO Dem Control Token

**Comment by Tedko** [comment]
@guanbinrui @zhouhanseng lets hard code and add these tokens

**Comment by Tedko** [comment]
> @guanbinrui @zhouhanseng lets hard code and add these tokens

for search enhance and $ticker hover func

**Comment by BenDavidow** [comment]
$yKYC
Token Address: 0x14de81C71B3F73874659082b971433514E201B27
Explorer: [Etherscan](https://etherscan.io/address/0x14de81C71B3F73874659082b971433514E201B27)
Website: [Foundry.Finance](https://foundry.finance/)
Community: [Twitter](https://twitter.com/catnip_exchange), [Discord](https://discord.gg/dBN2tHZ)
Source Code: [Github](https://github.com/aug-dao/augur_foundry)
Tags: Prediction Markets

**Comment by zhouhanseng** [comment]
0x api does not expose the price of those tokens yet. I can integrate balancer which provides info for them.

**Comment by guanbinrui** [comment]
Closed by #2092
